docs/xbcloud/xbcloud-iam-profile.md

```diff
@@ -0,0 +1,57 @@
+# Xbcloud with an IAM instance profile
+
+An authentication system has two elements:
+
+* Who am I?
+* What can I do?
+
+A role defines "what can I do." A role provides a method to define a collection of permissions. Roles are assigned to users, services and EC2 instances, the "who am I" element.
+
+The IAM instance profile is the "who" for an EC2 instance and assumes the IAM role, which has permissions. The instance profile has the same name as the IAM role.
+
+An IAM instance profile does not need the `-s3-secret-key` if they are running `xbcloud` from an Amazon EC2 instance. To configure or attach an instance metadata to an EC2 instance, see [How can I grant my Amazon EC2 instance access to an Amazon Se bucket](https://aws.amazon.com/premiumsupport/knowledge-center/ec2-instance-access-s3-bucket/). 
+
+An example of the command:
+
+```text
+$ xtrabackup ... | xbcloud put --storage=s3 --s3-bucket=bucket-name backup-name
+```
+
+The xbcloud binary outputs a "using metadata for access and secret key" to connect message.
+
+??? example "Expected output"
```

Comment:
what is those ???

Reply:
It allows us to hide the expected output.